docs/api-reference/cli.md

```diff
@@ -87,6 +87,10 @@ You can also set the hostname to be different from the default of `0.0.0.0`, thi
 ```bash
 npx next dev -H 192.168.1.2
 ```
+If you would like to view your app on your phone. We suggest setting your local ip as your host.
+```bash
+next dev -H $(ipconfig getifaddr en0)
```

Comment:
```suggestion
npx next dev -H $(ipconfig getifaddr en0)
```

Reply:
Does this work on Windows?

Reply:
No, this is macOS specific... Also doesn't work if you use a different interface, for example WLAN (en1).

Reply:
Then I don't think this is worth documenting because it will be confusing when it doesn't work as expected